src/ekobox/evol_pdf.py

```diff
@@ -111,4 +113,7 @@ def ekofileid(theory_card, operators_card):
     str
         file name
     """
-    return f"o{operators_card['hash'][:6]}_t{theory_card['hash'][:6]}.tar"
+    try:
+        return f"o{operators_card['hash'][:6]}_t{theory_card['hash'][:6]}.tar"
+    except KeyError:
+        return "o000000_t000000.tar"
```

Comment:
Wait, where does this hard-coded string come from?

Reply:
Maybe re-raising a more explicit error would be more appropriate...

Reply:
The problem is that in the new format we don't have `hash` anymore. So, even in this case, we need to choose how to fix. This was a temporary solution just to have tests working but of course we need to make a decision.